### ISSUE TITLE
Why limit to single thread?

### ISSUE THREAD

**Issue by linkerlin** [facebookresearch/denoiser#15]
enhance.py
torch.set_num_threads(1)

``` 
def get_estimate(model, noisy, args):
    torch.set_num_threads(1)
    if args.streaming:
        streamer = DemucsStreamer(model, dry=args.dry)
        with torch.no_grad():
            estimate = torch.cat([
                streamer.feed(noisy[0]),
                streamer.flush()], dim=1)[None]
    else:
        with torch.no_grad():
            estimate = model(noisy)
            estimate = (1 - args.dry) * estimate + args.dry * noisy
    return estimate

**Comment by adefossez** [comment]
Because this is run in parallel in many workers (by default 10 I think) so there is no point in having each worker use more than one thread.

**Comment by adefossez** [comment]
I'll be closing this issue now, feel free to open a new one if you still have questions.
